### PR TITLE
[MRG+2]: fix bad ch_names due to picks in get_peaks

### LINKED ISSUE #2724: Evoked.get_peak
Hi
I think there is a bug in Evoked.get_peak() function, when using ch_type :
In    : ev.get_peak('mag',tmin=tmin,tmax=tmax)
Out : ('MEG0122',0.178999)
ch_idx is related to data[picks] instead of self.data and the function should return 
self.ch_names[picks[ch_idx]] instead of self.ch_names[ch_idx]

### PR DESCRIPTION
**Pull request by agramfort**
closes #2724

### CODE REVIEW

**Comment by agramfort** [comment]
good to go at my end if travis is happy

**Comment by agramfort** [comment]
travis is happy...

File: doc/whats_new.rst [hunk 1]
```diff
@@ -51,6 +51,8 @@ BUG
 
     - Added lower bound in :func:`mne.stats.linear_regression` on p-values ``p_val`` (and resulting ``mlog10_p_val``) using double floating point arithmetic limits by `Eric Larson`_
 
+    - Fix channel name pick in ``get_peak`` method of Evoked by `Alex Gramfort`_
```
Comment (Eric89GXL) [inline doc/whats_new.rst]:
```
:class:`mne.Evoked`
```

**Comment by Eric89GXL** [comment]
Thanks @agramfort I'll fix that in `master`

**Comment by agramfort** [comment]
cherry pick 85a432de26427bfcdfeacf9e056cd01e7ea41a95

**Pull request merged**